src/kf_scripts/multiply_posteriors.py

```diff
@@ -0,0 +1,51 @@
+import xarray as xr
+import os
+
+
+def multiply_posteriors(period_number, base_directory):
```

Comment:
I wonder if we should add an additional period dimension to the dynamic scale factor file so that we can have a record of the scale factors used at each stage. Maybe this is already being done?

Reply:
After discussion, we decided it's best to retain current behavior of saving prior and posterior scale factors for each period to an `archive_sf` directory.